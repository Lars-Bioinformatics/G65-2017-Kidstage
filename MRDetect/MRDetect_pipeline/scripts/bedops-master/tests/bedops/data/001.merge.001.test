chr7	2	101
chr7	3	58
chr7	58	102
chr7	105	111
chr7	105	112
chr7	111	114
chr7	112	115
chr7	115	116
chr7	117	151
