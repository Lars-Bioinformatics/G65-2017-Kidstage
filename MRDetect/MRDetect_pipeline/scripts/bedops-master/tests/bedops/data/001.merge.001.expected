chr7	2	102
chr7	105	116
chr7	117	151
